zeus/client.py

```diff
@@ -23,14 +23,17 @@
 
 class ZeusClient(MetricsInterface, LogsInterface, AlertsInterface,
                  TrigAlertsInterface):
-
     """Zeus Client class, implementing wrapper methods for the
     Zeus API.
 
     """
 
     def __init__(self, user_token, server):
         self.token = user_token
```

Comment:
@travelist 
user_token is duplicated, but for compatibly, it's necessary, right?
(Just asking)